Setup.sh

```diff
@@ -76,7 +76,7 @@ spatial package retrieve schema      standard_library                        "${
 spatial package retrieve worker_sdk  c_headers                               "${PINNED_CORE_SDK_VERSION}"   ${DOMAIN_ENVIRONMENT_VAR:-}   "${CORE_SDK_DIR}"/worker_sdk/c_headers.zip
 spatial package retrieve worker_sdk  c-dynamic-x86_64-clang-macos            "${PINNED_CORE_SDK_VERSION}"   ${DOMAIN_ENVIRONMENT_VAR:-}   "${CORE_SDK_DIR}"/worker_sdk/c-dynamic-x86_64-clang-macos.zip
 
-if [[ -v DOWNLOAD_MOBILE ]];
+if [[ -n DOWNLOAD_MOBILE ]];
```

Comment:
this will now always download the mobile packages. We want to only download them, if we run `Setup.sh --mobile`.

```suggestion
if [[ -n "${DOWNLOAD_MOBILE}" ]];
```

that should fix it

Reply:
I think `"` is not necessary here when using `[[]]`. I tried this just now will get another error:
Setup.sh: line 79: DOWNLOAD_MOBILE: unbound variable

Reply:
If really want to use "${DOWNLOAD_MOBILE}",  maybe should use `[]`.

Reply:
ah sorry, you also need to add this line 
`DOWNLOAD_MOBILE=`  
after line 21.

This makes sure the variable is not unbound and it should work as expected. Using " makes sure that, should there ever be space in there, it will get evaluated correctly. This is the coding standard that we have across the company and also passes most, if not all, lint checkers